.gitignore

```diff
@@ -123,3 +123,4 @@ iotdb-core/tsfile/src/main/antlr4/org/apache/tsfile/parser/gen/
 # Develocity
 .mvn/.gradle-enterprise/
 .mvn/.develocity/
+/iotdb-core/ainode/poetry.lock
```

Comment:
It seems that the dependabot required a poetry.lock file to create PR. Should we add it?
https://github.com/apache/iotdb/security/dependabot/294

Reply:
I just had a look at the PLC4X plc4go module, we do check in the go.sum file (which is something similar) ... I guess another type of project worth having a look would be node-based ones.

But sonsidering the volatility of versions in that space (possibly also in python space) it might be a good idea to check in the file as well.

Reply:
Just re-read your comment ... I just added the poetry.lock file and removed it from the gitignore.